core/app/models/mno_enterprise/audit_event.rb

```diff
@@ -1,5 +1,8 @@
 module MnoEnterprise
   class AuditEvent < BaseResource
+    property :created_at, type: :time
```

Comment:
Yes. I did not find a way to declare it automatically on all the class. Adding it to base_resource does not work, as it only populate the attributes of base_resource, not the child class.